### ISSUE TITLE
Kravkedja på fakturor

### ISSUE THREAD

**Issue by frofrik** [equipe/equipe_issues#115]
Lås inte systemet direkt efter förfallodatum. Förslagsvis kan kravkedjan se ut nånting i stil med:

Dag 1: faktura skickas ut
Dag 30: faktura förfaller
Dag 35: påminnelse
Dag 45: systemet låser sig

**Comment by jnstq** [comment]
Yes, vi skickar redan påminnelser, vi ska bara ha lite flex i due on i app
